providers/os/resources/asset_eol.go

```diff
@@ -54,51 +83,51 @@ func initPlatformEol(runtime *plugin.Runtime, args map[string]*llx.RawData) (map
 
 // ^^
 
-func initAssetEol(runtime *plugin.Runtime, args map[string]*llx.RawData) (map[string]*llx.RawData, plugin.Resource, error) {
-	conn := runtime.Connection.(shared.Connection)
-	platform := conn.Asset().Platform
-	eolPlatform := convertPlatform2VulnPlatform(platform)
+// FIXME: DEPRECATED, update in v10.0 vv
```

Comment:
Do we need to add a note in the lr file? For `platform.eol`, we have:
```
// Information about the platform end-of-life. Deprecated in favor of `asset.eol`. This resource will be removed in version 10.0.
// Please use asset.eol instead.
```

Reply:
We have that already added to platform eol resource.